bigchaindb/consensus.py

```diff
@@ -162,6 +171,14 @@ def validate_transaction(bigchain, transaction):
                     raise exceptions.DoubleSpend(
                         'input `{}` was already spent'.format(fulfillment['input']))
 
+                txids.append(fulfillment['input']['txid'])
```

Comment:
Seems like a bit of a waste of effort if we've just got the transaction before on https://github.com/bigchaindb/bigchaindb/pull/599/files#diff-d82bc8f43aa0fddab154084c829f426aR160